### ISSUE TITLE
Jungle grass

### ISSUE THREAD

**Issue by everamzah** [BlockMen/Wasteland#7]
Is it intended that Jungle Grass still generates in some areas?

**Comment by BlockMen** [comment]
Fixed, thx. Just being curious, have you seen it often? I can't remember to have seen it once.

**Comment by ghost** [comment]
I saw it only once, didn't want to venture onward in case it was a mistake.

**Comment by ghost** [comment]
I didn't think of it initially, but I assumed the sheets of snow on dirt and stone were okay?

**Comment by ghost** [comment]
![screenshot_20151220_160137](https://cloud.githubusercontent.com/assets/1761623/11920058/0ed32c94-a733-11e5-924c-73941f3f80fe.png)

**Comment by ghost** [comment]
I'm on mgv6, so I have now set mgv6_flags = nojungles, nosnowbiomes.  I hope this helps.

**Comment by ghost** [comment]
![screenshot_20151220_162923](https://cloud.githubusercontent.com/assets/1761623/11920223/e5068b82-a736-11e5-8b9c-1ff58d8a5172.png)

Now I don't even know, and I guess I'll comment that back out the way it was...  Advice?

**Comment by ghost** [comment]
If mg_flags = notrees, then why would mgv6_spflags = trees?  Are these two different things?  I found this in map_meta:
mg_flags = notrees, caves, nodungeons, noflat, light, decorations
mgv6_spflags = jungles, biomeblend, mudflow, snowbiomes, noflat, trees

**Comment by BlockMen** [comment]
Ah, yes. There was an engine change with decorations and trees concerning mgv6_spflags. I knew it will cause trouble. @paramat !

Try:

mgv6_spflags = nojungles, biomeblend, mudflow, snowbiomes, noflat, notrees
mg_flags = notrees, caves, nodungeons, noflat, light, decorations

And what was you wanted to know concerning snow?

**Comment by ghost** [comment]
I was just wondering if the snow was part of the Wasteland theme, meaning whether it was supposed to be there.  Personally, I think it's nice and added a recipe of 6 snowballs and 1 empty bucket for water.

**Comment by BlockMen** [comment]
> I was just wondering if the snow was part of the Wasteland theme, meaning whether it was supposed to be there.

Snowcaps yes, snow biomes, no.

> Personally, I think it's nice and added a recipe of 6 snowballs and 1 empty bucket for water.

Ok, what version are you using (or what mods do you use?), since snow + buckets = water is implemented since 2 years: https://github.com/BlockMen/Wasteland/commit/1d9576e56dac1fb2e19a8d8096c55ad0736a3a0b

**Comment by ghost** [comment]
:O well color me surprised!  If I'd known, I wouldn't have added a duplicate recipe. ;) Ha!  Thanks for that.

**Comment by paramat** [comment]
>  I found this in map_meta:
> mg_flags = notrees, caves, nodungeons, noflat, light, decorations
> mgv6_spflags = jungles, biomeblend, mudflow, snowbiomes, noflat, trees

Yes sorry, it looks messy but this is the only way to retain backwards compatibility.
Both 'flat' and 'trees' flags have now been moved into 'mgv6_spflags', the flags still appear in 'mg_flags' due to compatibility but should be ignored, and yes their settings will often contradict the settings in 'mgv6_spflags'.

> mgv6_flags = nojungles, nosnowbiomes

Should be:
mgv6_spflags = nojungles, nosnowbiomes
Snowbiomes are enabled by default in mgv6 so you do need to disable them.

**Comment by ghost** [comment]
Very interesting, and sorry for any confusion stemming from my typo.

**Comment by ghost** [comment]
whoops pressed the wrong button
